lib/src/Uncertainty/StatTests/VisualTest.cxx

```diff
@@ -353,6 +364,13 @@ GridLayout VisualTest::DrawPairsMarginals(const Sample & sample, const Distribut
       const Indices indices = {j, i};
       const Cloud cloud(sample.getMarginal(indices), "blue", "fsquare", "");
```

Comment:
```suggestion
      const Cloud cloud(sample.getMarginal(indices));
```

Reply:
This is new graph :

![image](https://github.com/user-attachments/assets/d60d6933-4980-4d0a-a70a-66c7141758a2)